rules/windows/process_creation/proc_creation_win_certutil_decode.yml

```diff
@@ -0,0 +1,35 @@
+title: File Decoded From Base64/Hex Via Certutil.EXE
+id: cc9cbe82-7bc0-4ef5-bc23-bbfb83947be7
+status: test
+description: Detects execution certutil with either the "decode" or "decodehex" flags to decode base64 or hex encoded files. This can be abused by attackers to decode an encoded payload before execution
```

Comment:
```suggestion
description: Detects the execution of certutil with either the "decode" or "decodehex" flags to decode base64 or hex encoded files. This can be abused by attackers to decode an encoded payload before execution
```